### PR TITLE
addressed the issue #3

### PR DESCRIPTION
**Pull request by youngminoh7**
did the editing, please check.

### CODE REVIEW

File: new_file_two.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 Added by owner
+
+Edited by the developer
```
Comment (neuralintlab) [inline new_file_two.txt:3]: Yet, add another line.

File: new_file_two.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 Added by owner
```
Comment (neuralintlab) [inline new_file_two.txt:1]: Here, add you as well

File: new_file_two.txt (hunk 1 above)
Comment (youngminoh7) [reply new_file_two.txt:3]: OK

File: new_file_two.txt (hunk 2 above)
Comment (youngminoh7) [reply new_file_two.txt:1]: OK

**Review by neuralintlab — CHANGES_REQUESTED** [review]
Change this

**Review by neuralintlab — APPROVED** [review]

**Pull request merged**